client/src/components/BoxChat/BoxChat.js

```diff
@@ -65,9 +67,19 @@ export default {
 		this.emitter.on('on-transfer-room-data', async (roomData) => {
 			this.roomName = roomData.currentRoomName;
 			this.roomId = roomData.currentRoomId;
+			this.messages = [];
+
 			try {
 				let response = await ApiService.getMessageOfRoom(this.roomId);
-				console.log('Response:', response);
+				console.log('All message API response:', response);
+				let allMessData = response.data;
+
+				for (let element of allMessData) {
+					this.messages.push({
+						mess: element.message,
+						userId: element.user_id,
+					});
```

Comment:
- Đối với việc fetch data từ bên server thì không nên push trực tiếp từng element vào ```this.message``` tránh cho DOM ảo của Vuejs chạy nhiều lần
- Nên tạo 1 biến array để lưu tạm, rồi gán this.messges bằng array đó
--> Cũng k cần làm rỗng ```this.messages = [];```

Reply:
done